cms/bundles/checks.py

```diff
@@ -0,0 +1,60 @@
+from collections.abc import Sequence
+from typing import Any, Optional
+
+from django.apps import AppConfig
+from django.conf import settings
+from django.core.checks import CheckMessage, Info, register
+from django.core.checks import Warning as DjangoWarning
+
+from cms.bundles.api import BundleAPIClient, BundleAPIClientError
+
+
+@register()
+def check_bundle_api_health(app_configs: Optional[Sequence[AppConfig]], **kwargs: Any) -> list[CheckMessage]:  # pylint: disable=unused-argument
+    """Check if the Bundle API is healthy when enabled."""
+    errors: list[CheckMessage] = []
+
+    # Only check if Bundle API is enabled
+    if not getattr(settings, "ONS_BUNDLE_API_ENABLED", False):
+        return errors
+
+    try:
+        client = BundleAPIClient()
+        response = client.get_health()
+
+        # Check if the response indicates healthy status
+        if response.get("status") == "disabled":
+            # API is disabled, which is expected when ONS_BUNDLE_API_ENABLED is False
+            return errors
```

Comment:
Will this ever happen?. We never get this far if the API isn't enabled as per:
https://github.com/ONSdigital/dis-wagtail/blob/f0ea3ed701eb8903a35f0be26fe5bf9dd166d3cb/cms/bundles/checks.py#L17-L19

Reply:
health check removed in a8cee1bfe5788d75a016ac63d22dc28583b7aab5